### ISSUE TITLE
スクリーンリーダーで読み上げる際に、<th>タグの範囲を正しく解釈し、読み上げる事ができない

### ISSUE THREAD

**Issue by masanori1102** [tokyo-metropolitan-gov/covid19#1922]
## 起こっている問題 / The Problem
スクリーンリーダーで読み上げる際に、`<th>`タグの範囲を正しく解釈し、読み上げる事ができない

sonarsource Rules: [HTML: "&lt;th&gt;" tags should have "id" or "scope" attributes](https://rules.sonarsource.com/html/RSPEC-1102)

## スクリーンショット / Screenshot
なし

## 期待する見せ方・挙動 / Expected Behavior
`<th>`タグに`scope="col"`を追加し、同じ列に対応することを関連付け、読み上げに対応する

資料: [scope属性 ≪ th要素 ≪ 表（テーブル） ≪ 要素 ≪ HTML5入門](http://html5.cyberlab.info/elements/table/th-scope.html)

## 起こっている問題の再現手段 / Steps to Reproduce
ソースコードをSonarCloudで静的解析する

## 動作環境・ブラウザ / Environment
- macOS / Windows / Linux / iOS / Android
- Chrome / Safari / Firefox / Edge / Internet Explorer

**Comment by masanori1102** [comment]
これ、やります
